### DISCUSSION TITLE
What is this setting regarding the Win 10 start menu?

### DISCUSSION THREAD

**Discussion by mustafatoktas** [valinet/ExplorerPatcher#3293] [Q&A]
🟢Rounded corners, floating menu
🟢Rounded corners, docked menu
What is the difference between these two settings?

![1](https://github.com/valinet/ExplorerPatcher/assets/52534609/605513fa-c36c-4705-a055-03f127220ff4)

**Comment by milestprower92** [comment]
Rounded corners, floating menu:
makes the Windows 10 start menu rounded and floating like the Windows 11 start menu

Rounded corners, docked menu:
makes the Windows 10 start menu rounded

Not rounded:

the original look of the windows 10 start menu

**Comment by mustafatoktas** [comment]
🟡First of all, thank you for your explanation @milesprower2293 
🟢I know the difference between not rounded and rounded corners. I can also see.
🟢I don't understand the difference between `floating menu` and  `docked menu`. Is there any chance you can explain it with a visual or a short video?

| **Not rounded** | **Rounded corners** |
|     --------------------       |     ------------------     |
|  ![1](https://github.com/valinet/ExplorerPatcher/assets/52534609/117c3bc1-87e8-4714-a89c-ff71c7660827) |  ![2](https://github.com/valinet/ExplorerPatcher/assets/52534609/dbbf7f5e-0014-4c3a-bc87-c9ecdb4e71cb) |
